streamparse/storm/bolt.py

```diff
@@ -12,6 +12,50 @@ class JavaBolt(Component):
     @classmethod
     def spec(cls, name=None, serialized_java=None, full_class_name=None,
              args_list=None, inputs=None, par=1, config=None, outputs=None):
+        """Create a :class:`JavaBoltSpec` for a Java Bolt.
+
+        This spec represents this Bolt in a :class:`~streamparse.Topology`.
+
+        You must add the appropriate entries to your classpath by editing your
+        project's ``project.clj`` file in order for this to work.
+
+        :param name:   Name of this Bolt.  Defaults to name of
+                       :class:`~streamparse.Topology` attribute this is assigned
+                       to.
+        :type name:    `str`
+        :param serialized_java:  Serialized Java code representing the class.
+                                 You must either specify this, or
+                                 both ``full_class_name`` and ``args_list``.
+        :type serialized_java: `bytes`
+        :param full_class_name: Full name of Java class, including namespaces.
```

Comment:
I suppose that "including namespaces" makes sense to a Python developer, but a Java developer would think of it as "fully qualified class name, i.e. include the package name".